docs: update README and tidy up docs (#365)
openebs/lvm-localpv: docs/quickstart.md

> 

Reviewer: ```suggestion
```

## Usage and Deployment

## Prerequisites

> Before installing the LocalPV-LVM driver please make sure your Kubernetes Cluster meets the following prerequisites:
> 1. All the nodes must have LVM2 utils package installed
> 2. All the nodes must have dm-snapshot Kernel Module loaded - (Device Mapper Snapshot)

> > 3. You have access to install RBAC components into `<OPENEBS>` namespace.

Reviewer: This is not needed, it's handled via helm right?

```suggestion
```

<BR>

> ## Setup

Reviewer: ```suggestion

## Setup
```

> Find the disk which you want to use for the LocalPV-LVM. Note: For testing you can use the loopback device.
> 
> ```
> truncate -s 1024G /tmp/disk.img
> sudo losetup -f /tmp/disk.img --show
> ```

Reviewer: ```suggestion
Find the disk which you want to use for the LocalPV-LVM.

> **Note**: For testing you can use a loopback device:
> ```bash
> truncate -s 1024G /tmp/disk.img
> sudo losetup -f /tmp/disk.img --show
> ```
```

> [!NOTE]
> - LocalPV-LVM will not provision the VG for the user <BR>
> - The required Physical Volumes(PV) and Volume Group(VG) names will need to be created and present beforehand.

Create the Volume group on all the nodes, which will be used by the LVM2 Driver for provisioning the volumes

> ```

Reviewer: ```suggestion
```bash
```

sudo pvcreate /dev/loop0
sudo vgcreate lvmvg /dev/loop0       ## here lvmvg is the volume group name to be created
```

## Installation

Install the latest release of OpenEBS LVM2 LocalPV-LVM driver by running the following command. Note: All nodes must be running the same version of LocalPV-LVM, LMV2, device-mapper & dm-snapshot.

**NOTE:** Installation using operator YAMLs is not the supported way any longer.  

> We can install the latest release of OpenEBS LVM driver by running the following command:

Reviewer: ```suggestion
We can install the latest release of OpenEBS LVM driver by running the following command:

```
